Formula/rist.rb

```diff
@@ -0,0 +1,37 @@
+class Rist < Formula
```

Comment:
Should this be called `librist` rather than just `rist`? It seems `rist` refers to a collection of projects.

Reply:
> Should this be called librist rather than just rist? It seems rist refers to a collection of projects.

I had the same original idea. However the other projects are variants for other languages (C++, rust, etc) and this one remains the base project. And it does not only contain the library but also four basic tools such as `ristreceiver` and `ristsender`.

More importantly, RIST is an alternative to SRT. The existing SRT package on Homebrew is named `srt`, it contains the libsrt and a few basic tools, exactly like this RIST package. Consequently, for the sake of consistency with its SRT alternative, I named it `srt`. Honestly, if the SRT package had been named `libsrt`, I would have named it `librist`.

I assume that other projects could be named `rist-cpp`, `rist-go`, `rist-rust`, `rist-python`, etc, and have a dependency on `rist`.

Reply:
[Everyone else who packages this](https://repology.org/project/librist/versions) calls it `librist`, though. The same isn't true for `srt`, where there's a mix: some call it `libsrt`, others call it `srt`.

We should call this `librist`, since that's the name of the project.

Reply:
Alright. So shall we create two packages, `librist` with the library and `rist` with the tools and depending on `librist`?
Having a package named `libXXX` containing executables in addition to a library sounds odd, isn't it?

Reply:
No need; we can call this one `librist` and package the library and tools together. There's nothing wrong with a `lib*` package also shipping binaries. Others do it: `libarchive`, `libgcrypt`, `libassuan`, `libbluray`, `libevent`, `libidn`, etc...

The Arch package is also still called `librist` despite also including the tools. Presumably the other packages are the same.

Reply:
I do not want to argue about that kind of details. The latest commit renamed the package to `librist`, keeping one single formula.